index.html

```diff
@@ -1,90 +1,77 @@
 <!DOCTYPE html>
 <html lang="en">
-  <head>
-    <meta charset="UTF-8" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
-
-    <!-- CSS -->
-    <link
-      rel="stylesheet"
-      href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css"
-    />
-    <link
-      rel="stylesheet"
-      href="https://use.fontawesome.com/releases/v5.8.1/css/all.css"
-      integrity="sha384-50oBUHEmvpQ+1lW4y57PTFmhCaXp0ML5d60M1M7uH2+nqUivzIebhndOJK28anvf"
-      crossorigin="anonymous"
-    />
-    <link
-      href="https://fonts.googleapis.com/css?family=Open+Sans&display=swap"
-      rel="stylesheet"
-    />
-    <link 
-      href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.8.3/font/bootstrap-icons.css"
-      rel="stylesheet" 
-    />
-    <link rel="stylesheet" href="./assets/css/style.css" />
-    <title>Umbrella</title>
-    <!-- Favicon -->
-    <link rel="shortcut icon" type="image/ico" href="./assets/images/favicon.ico">
-  </head>
-
-  <body>
-    <header class="bg-dark text-light text-center">
-      <h1>Umbrella</h1>
-      <p>An address book to find my friends, join them and see the current weather.</p>
-    </header>
-
-    <main>
-      <div class="d-flex flex-row">
-        <!-- Contacts section -->
-        <section id="contacts" class="d-flex flex-column bg-light col-5">        
-            <div class="d-flex flex-row">
-              <div class="col-11">
-                <h2 class= "bg-dark text-light">Contacts</h2>
-              </div>
-              <div class="col-1">
-                <button id="add-button">+</button>
-              </div>
-            </div>
-            <div>
-              <section id="contact-list">
-                <h3>Contact List</h3>
-                <ul>
-                <!-- href or buttons created from javascript function? AB/SW-->
-                </ul>
-            </section>
-            </div>
+
```

Comment:
Top half of these changes are just changes to indentation (the CSS links etc.) 
below <main> has changes to the containers and the column layouts